ui/components/MesheryPatterns.js

```diff
@@ -626,6 +626,29 @@ function MesheryPatterns({
         },
       },
     },
+    {
+      name : "visibility",
+      label : "Visibility",
+      options : {
+        filter : false,
+        sort : true,
+        searchable : true,
+        customHeadRender : function CustomHead({ index, ...column }) {
+          return (
+            <TableCell key={index}>
+              <b>{column.label}</b>
+            </TableCell>
+          );
+        },
+        customBodyRender : function CustomBody() {
+          return (
+            <>
+              <img src="/static/img/private.svg" />
```

Comment:
This should be dynamically referenced. Assuming all we have a logic defined to select the visibility. On that basis, we can show private or public. cc @gr455